### PR TITLE
feat(aave-v3): configure rewards action

### PR DESCRIPTION
**Pull request by tapakornl**

### CODE REVIEW

File: packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+export class ConfigureAssetsTxBuilder extends TxBuilder {
+  emissionManager: Address
+  rewardsConfigInputs: RewardsConfigInput[]
+  constructor(client: InfinitWallet, params: ConfigureAssetsParams) {
+    super(ConfigureAssetsTxBuilder.name, client)
+
+    this.emissionManager = getAddress(params.emissionManager)
+    this.rewardsConfigInputs = params.rewardsConfigInputs.map((input) => ({
+      ...input,
+      asset: getAddress(input.asset),
```
Comment (Kittikornl) [inline packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts:32]: Can we set this to the asset that isn't listed?

File: packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      reward: getAddress(input.reward),
```
Comment (Kittikornl) [inline packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts:33]: reward can be zero address?

File: packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+      transferStrategy: getAddress(input.transferStrategy),
```
Comment (Kittikornl) [inline packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts:34]: transferStrategy can be zero address?

File: packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      rewardOracle: getAddress(input.rewardOracle),
```
Comment (Kittikornl) [inline packages/modules/aave-v3/src/actions/subactions/tx-builders/emissionManager/configureAssets.ts:35]: rewardOracle can be zero address?

File: packages/modules/aave-v3/src/actions/configureRewards.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+import { validateActionData, zodAddress } from '@infinit-xyz/core/internal'
+
+import { ApproveSubAction, ApproveSubActionParams } from '@actions/subactions/approve'
+import { ConfigureAssetsSubAction } from '@actions/subactions/configureAssets'
+import { ConfigureAssetsParams, RewardsConfigInput } from '@actions/subactions/tx-builders/emissionManager/configureAssets'
+
+import type { AaveV3Registry } from '@/src/type'
+
+export const ConfigRewardsParamSchema = z.object({
+  emissionManager: zodAddress.describe(`Address of emission manager e.g. '0x123...abc'`),
```
Comment (mewpawee) [inline packages/modules/aave-v3/src/actions/configureRewards.ts:13]: use zodNoZeroAddress

**Review by Kittikornl — APPROVED** [review]

**Pull request merged**